app_dart/lib/src/model/ci_yaml/target.dart

```diff
@@ -89,6 +89,14 @@ class Target {
     return BatchPolicy();
   }
 
+  /// Get the tags from the defined properties in the ci.
+  ///
+  /// Return an empty list if no tags are found.
+  List<String>? get getTags {
```

Comment:
```suggestion
  List<String> get getTags {
```

Based on the last line of the dartdoc, it can return null.

Reply:
Ah for some reason my change is not here. I changed it to an empty list as that would be safer than returning null.